kratos/solving_strategies/strategies/solving_strategy.h

```diff
@@ -109,13 +111,20 @@ class SolvingStrategy
     ///@name Life Cycle
     ///@{
 
+    /**
+     * @brief Default constructor
+     */
+    explicit SolvingStrategy() : mpModelPart(nullptr)
+    {
+    }
```

Comment:
```suggestion
    explicit SolvingStrategy() { }
```

I would not initialize it if the value it has is already the default

Reply:
Idem, I don't trust compilers decissions. This is harmless

Reply:
what do you mean? The member is initialized